recipe/meta.yaml

```diff
@@ -34,12 +34,12 @@ test:
     - pymt_sedflux
 
 about:
-  home: https://github.com/mcflugen/pymt_sedflux
+  home: https://github.com/pymt-lab/pymt_sedflux
   license: MIT
   license_family: MIT
   license_file: LICENSE
   summary: sedflux components wrapped as PyMT plugins.
-  dev_url: https://github.com/mcflugen/pymt_sedflux
+  dev_url: https://github.com/pymt-lab/pymt_sedflux
```

Comment:
@mcflugen it is not too important but after a metadata change we recommend to re-render so the README is also updated. You can wait for a new release to do that.